File: Tyche.iOS/DataBet/Sources/DataBet/Domain/PoolGamblerBet.swift
import Foundation
import Core

public struct PoolGamblerBet: Codable {
    public let poolId: String
    public let gamblerId: String
    public let matchId: String
    public let homeTeamId: String
    public let homeTeamName: String
    public let awayTeamId: String
    public let awayTeamName: String
    public let matchScore: TeamScore<Int>?
    public let betScore: TeamScore<Int>?
    public let score: Int?
    public let matchDateTime: Date
    public let isLocked: Bool
}

File: Tyche.Android/data/bet/src/main/java/com/felipearpa/tyche/data/bet/domain/PoolGamblerBet.kt
package com.felipearpa.tyche.data.bet.domain

import com.felipearpa.tyche.core.type.TeamScore
import kotlinx.datetime.LocalDateTime

data class PoolGamblerBet(
    val poolId: String,
    val gamblerId: String,
    val matchId: String,
    val homeTeamId: String,
    val homeTeamName: String,
    val awayTeamId: String,
    val awayTeamName: String,
    val awayTeamScore: Int?,
    val matchScore: TeamScore<Int>?,
    val betScore: TeamScore<Int>?,
    val score: Int?,
    val matchDateTime: LocalDateTime,
    val isLocked: Boolean,
)
